src/dso_api/dynamic_api/remote/views.py

```diff
@@ -28,6 +29,8 @@ class RemoteViewSet(ViewSet):
 
     serializer_class = None
     endpoint_url = None
+    dataset_id = None
```

Comment:
Elke `request` heeft al een dataset (via https://github.com/Amsterdam/dso-api/blob/master/src/dso_api/dynamic_api/middleware.py#L9): `request.dataset`.
Dus deze regel is niet nodig.
Mischien we kan hetzelfde voor de `tabel` doen?

Reply:
Voor remote views zijn er geen modellen ,daarom wordt daarvoor de dataset niet gezet in de middleware

Reply:
Ah! Duidelijk!